### PR TITLE
First pull

### PR DESCRIPTION
**Pull request by AleksMnix**
Ok

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,19 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Jqute) [inline pom.xml:1]: Don't delete pom.file!

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.idea
+*.iml
+out
+.gitignore
+README.md
```
Comment (Jqute) [inline .gitignore]: Don't ignore README.md, .xml

Comment (AleksMnix) [reply .gitignore]: Ok

File: pom.xml (hunk 1 above)
Comment (AleksMnix) [reply pom.xml:1]: returned to place

**Comment by Jqute** [comment]
LGTM 🌵

**Pull request merged**